inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/transformation/relation/RightOuterJoinNodeRelationTest.java

```diff
@@ -37,10 +37,15 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * Tests for{@link RightOuterJoinNodeRelationTest}
+ * Tests for {@link RightOuterJoinNodeRelationTest}
```

Comment:
`RightOuterJoinNodeRelationTest` should be RightOuterJoinNodeRelation

Reply:
done